share/wake/lib/core/list.wake

```diff
@@ -49,6 +49,18 @@ export data List a =
 # ```
     (head: a), (tail: List a)
 
+# Create a singleton list from a value. This
+# can be used to slightly increase the niceness
+# of lists by avoid the Nil at the end.
+#
+# Examples:
+# ```
+#   10,                      # Create a singleton list containg just 10
```

Comment:
```suggestion
#   10,                      # Create a singleton list containing just 10
```

Reply:
Also, unrelated to the typo I just pointed out, I was wondering if it would make sense to _not_ actually use the unary comma for single-item lists as a style preference. Python has exactly this syntax for single-item tuples, and I think it's a very frequent accident to construct a single-item tuple when you weren't expecting to, especially when refactoring code and moving lines around.

Alternatively, maybe we could prefer the unary comma for singleton lists _but_ also recommend wrapping it with parentheses to make it more obvious? E.g. `(10,)`

Reply:
Yeah I'm ambivalent. Eventually we want to have square bracket syntax per the poll, this is just a small tide over.

Reply:
I agree we should request parens around it in review.